### DISCUSSION TITLE
How to have Image Caption and Image Credits in different html tags

### DISCUSSION THREAD

**Discussion by ninanoleto** [igordanchenko/yet-another-react-lightbox#291] [Q&A] [answered]
Hello! :wave: In my use case, I need an html tag for the image caption and another for the image credits, as they have different styling requirements. Is this currently possible? From what I've seen in the documentation there is only a `description` prop? Would I then need to create a whole separate custom slide to add image credits? :thinking:

**Comment by igordanchenko** [comment, marked as answer]
Both `title` and `description` accept the `ReactNode` type, so you can use any JSX markup.

**Reply by ninanoleto** [reply]
Oh didnt notice that! thanks for the quick reply!! :sparkles:
